### PR TITLE
Shipment notification

### PR DESCRIPTION
**Pull request by biomoz**
ShipmentController: Added show, edit and delete function for shipments.
Shipment Model: Fixed an error. (HasOne should be HasMany.)
Notifications: Added notification table to database and added a dropdown menu for it in the navigation bar. (When there is an update on the shipment the user that created the shipment will receive a notification.)
Shipment Pages: Updated the pages (index, set, requests pages) for them to look cleaner and added show page to fully show details of shipment. Also added edit page to edit the shipment.
Route: Updated route to be able to function with added pages and features.

Let me know if there are any problems with the PR.

### CODE REVIEW

**Comment by ThomasFrans** [comment]
Can you just format it. It makes it easier to read and merge 🙂 I'll merge it on Sunday.

**Comment by biomoz** [comment]
I formatted with Laravel pint with ./vendor/bin/pint command. 3 files had changes but no issues on functionality after changes. Everything is working. Hopefully it's fine now. Let me know if there's anything else.

**Comment by ThomasFrans** [comment]
`php artisan format`

**Comment by biomoz** [comment]
php artisan format doesn't seem to work: 

PS C:\xampp\htdocs\AD> php artisan format

   Symfony\Component\Process\Exception\RuntimeException 

  TTY mode is not supported on Windows platform.

  at vendor\symfony\process\Process.php:1017
    1013▕      */
    1014▕     public function setTty(bool $tty): static
    1015▕     {
    1016▕         if ('\\' === \DIRECTORY_SEPARATOR && $tty) {
  ➜ 1017▕             throw new RuntimeException('TTY mode is not supported on Windows platform.');
    1019▕
    1020▕         if ($tty && !self::isTtySupported()) {
    1021▕             throw new RuntimeException('TTY mode requires /dev/tty to be read/writable.');      

  1   app\Console\Commands\FormatCode.php:41

  2   app\Console\Commands\FormatCode.php:29
      App\Console\Commands\FormatCode::format_php_code()

Are you sure it's in the main? And also are you using Linux?

**Comment by ThomasFrans** [comment]
The command is now fixed upstream. If you merge main, you can run it.

**Comment by biomoz** [comment]
I have done it. Now everything should be fine.

File: routes/web.php [hunk 1]
```diff
@@ -119,7 +119,16 @@
...
+    Route::get('/shipments', [ShipmentController::class, 'index'])->name('shipments.index');
+    Route::get('/shipments/{shipment}/edit', [ShipmentController::class, 'edit'])->middleware(['auth', 'verified'])->name('shipments.edit');
+    Route::patch('/shipments/{shipment}', [ShipmentController::class, 'update'])->middleware(['auth', 'verified'])->name('shipments.update');
+    Route::delete('/shipments/{shipment}', [ShipmentController::class, 'destroy'])->middleware(['auth', 'verified'])->name('shipments.destroy');
+    Route::get('/shipments/{shipment}', [ShipmentController::class, 'show'])->name('shipments.show');
+
+    //Notification
+    Route::get('/markAsRead', function () {
+        auth()->user()->unreadNotifications->markAsRead();
+    });
```
Comment (ThomasFrans) [inline routes/web.php:131]: Move into the middleware group above and remove the individual middleware.

**Review by ThomasFrans — CHANGES_REQUESTED** [review]
There's one stylistic things I would change to keep the code clean for everyone.

**Comment by biomoz** [comment]
Deleted the middleware part from the page routes as you asked. Also added mark all as read function to the navigation bar.

File: routes/web.php (hunk 1 above)
Comment (biomoz) [reply routes/web.php]: I deleted the middleware parts from the routes since they are not really necessary I think.

**Review by ThomasFrans — APPROVED** [review]

**Pull request merged**